src/main/java/com/direwolf20/buildinggadgets/client/gui/CopyGUI.java

```diff
@@ -47,11 +48,9 @@ public void init() {
         this.x = width / 2;
         this.y = height / 2;
 
-        startPos = BGItems.gadgetCopyPaste.getStartPos(copyPasteTool);
-        endPos = BGItems.gadgetCopyPaste.getEndPos(copyPasteTool);
-
-        if (startPos == null) startPos = new BlockPos(0, 0, 0);
-        if (endPos == null) endPos = new BlockPos(0, 0, 0);
+        Region reg = GadgetCopyPaste.getSelectedRegion(copyPasteTool).orElse(Region.singleZero());
+        startPos = reg.getMin();
```

Comment:
I just want to ensure that this is still the same logic. I'm sure it is but your getMin would be the first block and your getMax would be the last block selected by the player?

Reply:
Yes